compass/run.py

```diff
@@ -144,11 +157,22 @@ def run_test_case():
     config.read(test_case.config_filename)
     test_case.config = config
 
+    if steps_to_run is None:
+        steps_to_run = config.get('test_case',
+                                  'steps_to_run').replace(',', ' ').split()
+
+    if steps_not_to_run is not None:
+        steps_to_run = [step for step in steps_to_run if step not in
```

Comment:
Check if step name specified is actually a valid step name or not.  (This should be done for both `--steps` and `--no-steps`.)